app/views/content/is-teaching-right-for-me/if-you-want-to-change-career.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Career change to teaching
 heading: How do I change to a career in teaching?
-subcategory: Qualifications and experience
+subcategory: Changing careers
```

Comment:
I think you need to update the nav order of `if-you-want-to-change-career` (and possibly all the other pages below it as a knock-on factor) as the subcategory order is now very odd:

![image](https://github.com/user-attachments/assets/9d1330c3-6017-4c79-8892-4d97da0a094f)

Reply:
Will fix!